io.github.milkshiift.GoofCord.yml

```diff
@@ -0,0 +1,77 @@
+app-id: io.github.milkshiift.GoofCord
+
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+
+command: startgoofcord
+separate-locales: false
+
+finish-args:
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --share=ipc
+  - --talk-name=org.freedesktop.Notifications # Discord push notifs
```

Comment:
```suggestion
```

https://docs.flatpak.org/en/latest/electron.html#sandbox-permissions

> Electron uses libnotify on Linux to provide desktop notifications. libnotify [since 0.8.0](https://gitlab.gnome.org/GNOME/libnotify/-/merge_requests/27) automatically uses the [notification portal](https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.Notification.html) when inside a sandboxed environment and --talk-name=org.freedesktop.Notifications is not required.